src/plugins/clips-executive/clips/wm-config.clp

```diff
@@ -14,9 +14,22 @@
   ?cf <- (confval (path ?path&:(str-prefix (str-cat "/clips-executive/specs/" ?spec "/parameters/") ?path))
 									(type ?type) (value ?value) (is-list ?is-list) (list-value $?list-value))
```

Comment:
I don't think you need `?cf` anymore, as you do `do-for-all-facts` on the RHS.
```suggestion
```

Reply:
Oh yea true, meant to remove that, good catch. Done in https://github.com/fawkesrobotics/fawkes/pull/224/commits/edf6b737298443240e891af41292e5495e7ddaa6